flipperdevices/flipperzero-firmware
# [FL-185] Gauge calibration

# What's new

- Fix control command and add CEDV parameters configuration in bq27220 driver and api-hal-power
- Add design capacity print in power_test CLI command

# Verification

- Build for f4 and f5 targets
- Verify that design capacity changes to 2100mAh. Design capacity is printed in power_test CLI command

# Checklist (do not modify)
